src/types.ts

```diff
@@ -698,12 +698,12 @@ export type BanUserOptions<UserType = UR> = UnBanUserOptions & {
 };
 
 export type ChannelOptions = {
-  last_message_ids?: { [key: string]: string };
+  connection_id?: string;
```

Comment:
API infers the connection_id from the URL
```suggestion
```

Reply:
Removed `connection_id`